### ISSUE TITLE
Applying aria-label to certain empty elements generates an "attributes do not match their roles" error

### ISSUE THREAD

**Issue by PikeUK** [GoogleChrome/lighthouse#12630]
Applying aria-label to certain empty elements (e.g. i, b, span, div, etc.) generates an "[aria-\*] attributes do not match their roles" error in Lighthouse 8.0.0 (this problem doesn't occur in 7.2.0). This does not affect other empty elements (e.g. h1, section, form, etc.) so I assume it only affects elements without a default role? I haven't investigated whether this affects any other aria-* attributes.

As an example the following will generate an accessibility error:
`<i class="icon-search" aria-label="Search"></i>`

WAI-ARIA 1.1 describes aria-label as "Used in Roles: All elements of the base markup". Perhaps I'm misinterpreting it but that implies the role should be irrelevant for aria-label.

IMO this is a quite damaging bug as it encourages authors to remove accessibility labels. Icon fonts especially usually use empty i or span elements and can benefit from an aria-label.

**Comment by patrickhulce** [comment]
Thanks for filing @PikeUK!

That example seems like a valid error (shouldn't the icon have `role=image`?). See [Using the aria-label attribute on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute#support) for more on which elements are *actually supported cross-browser with screen readers*. 

If you believe there is still a bug here not covered in your example, please feel free to file in the [axe-core](https://github.com/dequelabs/axe-core/) library and Lighthouse will pickup any potential fix on the next update.

**Comment by PikeUK** [comment]
@patrickhulce Good point, I think you're correct. I'll file in axe-core if I find any other examples.
